packages/palette/src/elements/Avatar/Avatar.tsx

```diff
@@ -1,10 +1,15 @@
 import { TextVariant } from "@artsy/palette-tokens/dist/typography/v3"
 import React from "react"
+import styled from "styled-components"
 import { splitBoxProps } from "../Box"
 import { Flex, FlexProps } from "../Flex"
 import { Image, ImageProps } from "../Image"
 import { Text } from "../Text"
 
+const Container = styled(Flex)`
+  -webkit-mask-image: -webkit-radial-gradient(white, black);
```

Comment:
Wow, would love a comment here explaining the problem and why this fixed it though!

Reply:
Interesting: https://gist.github.com/ayamflow/b602ab436ac9f05660d9c15190f4fd7b?permalink_comment_id=3941712#gistcomment-3941712 — I find this comment a bit more helpful but I'm wondering if I can also track down an actual bug report in the Webkit bug tracker.